lib/ex_doc/formatter/html.ex

```diff
@@ -213,7 +213,7 @@ defmodule ExDoc.Formatter.HTML do
 
   defp build_api_reference(nodes_map, config) do
     api_reference = Templates.api_reference_template(config, nodes_map)
-    %{id: "api-reference", title: "API Reference", group: "", content: api_reference}
+    %{id: "api-reference", title: "<%= translate(config, "API Reference") %>", group: "", content: api_reference}
```

Comment:
@alchexmist Sorry it had just submitted directly to your repo, I thought it would show up as a suggestion, but it went right through